src/main/java/toughasnails/temperature/BlockTemperatureData.java

```diff
@@ -0,0 +1,14 @@
+package toughasnails.temperature;
+
+public class BlockTemperatureData {
+	public String blockName;
+	public float blockBaseTemperature;
+	public BlockTemperatureStateData[] stateTemperatures;
```

Comment:
Ok, I will refactor it to store native blockstates.

I will use a copy of your BOP config to make a TAN config and address this comment and moving the json out of GamplayConfigurationHandler.